Foundation/Thread.swift

```diff
@@ -256,7 +256,11 @@ open class Thread : NSObject {
         let maxSupportedStackDepth = 128;
         let addrs = UnsafeMutablePointer<UnsafeMutableRawPointer?>.allocate(capacity: maxSupportedStackDepth)
         defer { addrs.deallocate(capacity: maxSupportedStackDepth) }
+#if os(Android)
+        let count = maxSupportedStackDepth
```

Comment:
The count here is the number of successful frames returned by `backtrace`, and that they've been filled in by the caller. Since they won't have been any returned, isn't 0 the correct answer in the face of not being implemented? Admittedly this is being stubbed out from the caller below, but if it gets added elsewhere later then it would be better if this didn't contain a known bad number.